Storage/Storage/Tools/StorageType+Extensions.swift

```diff
@@ -537,4 +537,22 @@ public extension StorageType {
     func loadPaymentGatewayAccount(siteID: Int64, gatewayID: String) -> PaymentGatewayAccount? {
         let predicate = \PaymentGatewayAccount.siteID == siteID && \PaymentGatewayAccount.gatewayID == gatewayID
         return firstObject(ofType: PaymentGatewayAccount.self, matching: predicate)
-    }}
+    }
+
+    // MARK: - System plugins
+
+    /// Returns all stored system plugins for a provided `siteID`.
+    ///
+    func loadSystemPlugins(siteID: Int64) -> [SystemPlugin] {
+        let predicate = \SystemPlugin.siteID == siteID
+        let descriptor = NSSortDescriptor(keyPath: \SystemPlugin.name, ascending: true)
```

Comment:
[Q] Is there any reason to sort them by name?

Reply:
Nope. I did it because maybe in the future we need to list it in the correct order, but I can remove it since we don't need it yet.

Reply:
We can leave it, I was just curious about the intention. 
FYI we don't have to show this list anywhere for now...